### PR TITLE
Feature/enhancement2401

### PR DESCRIPTION
**Pull request by mehab**
### Description

Backported changes from issue https://github.com/DependencyTrack/dependency-track/issues/2401

### Checklist

- [x] I have read and understand the [contributing guidelines](../CONTRIBUTING.md#pull-requests)
- [ ] This PR fixes a defect, and I have provided tests to verify that the fix is effective
- [x] This PR implements an enhancement, and I have provided tests to verify that it works as intended
- [ ] This PR introduces changes to the database model, and I have added corresponding [update logic](https://github.com/DependencyTrack/dependency-track/tree/master/src/main/java/org/dependencytrack/upgrade)
- [ ] This PR introduces new or alters existing behavior, and I have updated the [documentation](https://github.com/DependencyTrack/dependency-track/tree/master/docs/_docs) accordingly

### CODE REVIEW

File: .run/Build API Server Image.run.xml
Comment (nscuro) [inline .run/Build API Server Image.run.xml:1]: This change seems unrelated?

File: .run/Build Bundled Image.run.xml
Comment (nscuro) [inline .run/Build Bundled Image.run.xml:1]: This change seems unrelated?

**Review by nscuro — CHANGES_REQUESTED** [review]

File: .run/Build Bundled Image.run.xml
Comment (mehab) [reply .run/Build Bundled Image.run.xml:1]: yes.. I reverted it but now there is a change in indentation

**Review by nscuro — APPROVED** [review]
